lib/MusicBrainz/Server/Controller/Tag.pm

```diff
@@ -21,6 +21,16 @@ sub _load
     return $c->model('Tag')->get_by_name($name);
 }
 
+sub no_tag_provided : Path('/tag')
```

Comment:
I think `Args(0)` is implied there (I guess it must be if it works, but can't find any relevant docs), but just for clarity it'd be nice to add `Args(0)` after the `Path` part to indicate it's the end of the path.

Reply:
I think Path only works for the full path, otherwise it'd be PathPart?

Reply:
Or actually maybe not, since /tags/whatever leads to the tag cloud (wonder if it should actually redirect to /tag/whatever, heh). Ok, I'll change it.